types/use-position/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for use-position 0.0
+// Project: https://github.com/trekhleb/use-position
+// Definitions by: Claudio Musumeci <https://github.com/muscla87>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace UsePosition {
```

Comment:
Is there a need for the namespace? You can just export `PositionOptions` at the top level. See more at this long-winded explanation https://stackoverflow.com/a/30357635/4386952

Reply:
Thank you for your review. I have removed the unnecessary namespace.